Resources/Prototypes/Body/Parts/prawn.yml

```diff
@@ -0,0 +1,231 @@
+# TODO: Add descriptions (many)
+# TODO BODY: Part damage
```

Comment:
more copy and pasted comments. Anything with a `#` in front of it that you didn't write probably shouldn't be there.